### PR TITLE
Add rules for stale bot

### LINKED ISSUE #2188: Dealing with old issues
We have a lot of legacy tickets - some feature requests, others issues that cannot be reproduced. To help reduce the noise and to keep our issues under control how about using some of the bots that are available on the marketplace such as: https://github.com/apps/stale

Opinions?

### PR DESCRIPTION
**Pull request by inverse**
Resolves #2188

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
```
Comment (inverse) [inline .github/stale.yml]: Happy to remove these comments - or do you think they help explain?

Comment (alexesprit) [reply .github/stale.yml]: I think it's quite enough to understand the config w/o comments - properties are named good.

Comment (inverse) [reply .github/stale.yml]: Removing :)

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - feature
+  - request
+# Label to use when marking an issue as stale
+staleLabel: wontfix
```
Comment (alexesprit) [inline .github/stale.yml]: Maybe use `outdated` label? I created `wontfix` label to mark issue that won't be resolved due to some reasons.

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
```
Comment (alexesprit) [inline .github/stale.yml]: I'd add a new line to make GitHub happy :D

Comment (inverse) [reply .github/stale.yml]: rly need to change my editor to auto do this :/

**Comment by alexesprit** [comment]
Looks good for me. If you have no additional changes, we can try to use this config.

**Pull request merged**